src/System.ServiceProcess.ServiceController/tests/System.ServiceProcess.ServiceController.TestService/TestService.cs

```diff
@@ -74,6 +74,8 @@ protected override void OnShutdown()
 
         protected override void OnStart(string[] args)
         {
+            File.Delete(GetLogPath(ServiceName));
```

Comment:
Are there cases in normal test flow where the file already exists and needs to be deleted?

Reply:
Yes, two tests that pass arguments start and stop the service first, because newing up the TestServiceProvider() starts it automatically.